constants/user.py

```diff
@@ -33,3 +33,6 @@
 
 account_list_item = namedtuple('AccountListItem', ['name', 'color', 'emoji'])
 wallet_account_list_item = namedtuple('WalletAccountListItem', ['name', 'icon', 'object'])
+
+wallet_account = namedtuple('PrivateKeyAddressPair', ['private_key', 'wallet_address'])
+privatekeyaddresspair1 = wallet_account('2daa36a3abe381a9c01610bf10fda272fbc1b8a22179a39f782c512346e3e470', '0xd89b48cbcb4244f84a4fb5d3369c120e8f8aa74e')
```

Comment:
Would be nice to add underscore for privatekeyaddresspair1  too to make it more readable, what do you think?

Reply:
ah, sure, u are definitely right